### PR TITLE
newsletter 07-19-2019

### PR DESCRIPTION
**Pull request by ryan-shea**
@coderberry waiting on Solana bit.ly tracker link until merge

@gitcoinco/engineers pls wait until we patch the URL into the email!

### CODE REVIEW

File: app/retail/emails.py [hunk 1]
```diff
@@ -932,8 +932,8 @@ def render_start_work_applicant_expired(interest, bounty):
 def render_new_bounty_roundup(to_email):
     from dashboard.models import Bounty
     from django.conf import settings
-    subject = "The Final Hours of Beyond Blockchain ⛓"
-    new_kudos_pks = [4144, 4141, 4143]
+    subject = "Grow Ethereum Draws Nearer"
```
Comment (owocki) [inline app/retail/emails.py]: should we call it `Grow Ethereum Hackathon`?  i worry ppl wont know what 'Grow Ethereum' is unless we specify..

File: app/retail/emails.py [hunk 2]
```diff
@@ -948,11 +948,10 @@ def render_new_bounty_roundup(to_email):
 Hey Gitcoiners,
 </p>
 <p>
-The Beyond Blockchain hackathon is coming to a close. Thank you to all of our participants that hacked along during the past three weeks; we're ecstatic to see what you all have created. We will be releasing another retro in the coming days as winners are announced.
+Grow Ethereum is right around the corner -- only 11 days remain until the hacking begins❗️ We are expecting great sponsors, including confirmation from the Ethereum Foundation itself (!) and UNICEF. Build projects together with top Ethereum companies and enterprises, win crypto, and unleash a new era of decentralized global infrastructure. The link to register is <a href="https://hackathons.gitcoin.co/grow-ethereum/" target="_blank">here.</a> 🌱
```
Comment (owocki) [inline app/retail/emails.py]: same comment here

> should we call it `Grow Ethereum Hackathon`?  i worry ppl wont know what 'Grow Ethereum' is unless we specify..

File: app/retail/emails.py [hunk 3]
```diff
@@ -973,67 +972,46 @@ def render_new_bounty_roundup(to_email):
...
-        'primer': 'Crypto-Market Application using Bluzelle',
+        'url': 'https://github.com/RibbonBlockchain/IncentivesMVP/issues/8',
+        'primer': 'Enable camera on front end for capture passport on patient and practitioner onboarding',
     }, {
-        'url': 'https://github.com/gitcoinco/web/issues/4745',
-        'primer': 'Allow Ability To Resurface Bounties To The Top When Bounty Has Been Abandoned',
+        'url': 'https://github.com/centrifuge/precise-proofs/issues/88',
+        'primer': 'Optimize the fixed height tree implementation',
 }, ]
-
```
Comment (owocki) [inline app/retail/emails.py]: did you mean to remove the leaderboard?

File: app/retail/emails.py (hunk 1 above)
Comment (ryan-shea) [reply app/retail/emails.py]: good point. i'll update across the newsletter

File: app/retail/emails.py (hunk 3 above)
Comment (ryan-shea) [reply app/retail/emails.py]: nope. my bad.

**Comment by ryan-shea** [comment]
@owocki fixed in above commit

@danlipert ready to ship

**Pull request merged**